source/documentation/virtual_machine_management_guide/topics/Installing_usbdk_on_Windows.adoc

```diff
@@ -23,6 +23,17 @@ https://_[your manager's address]_/ovirt-engine/services/files/spice/usbdk-x86.m
 ----
 https://_[your manager's address]_/ovirt-engine/services/files/spice/usbdk-x64.msi
 ----
+endif::rhv-doc[]
+
+ifdef::ovirt-doc[]
+* `usbdk` download page:
```

Comment:
Change to `* link:https://www.spice-space.org/download.html[`usbdk` download page]`